### ISSUE TITLE
metadata not included in backup

### ISSUE THREAD

**Issue by agent47git** [alekdavis/PlexBackup#90]
Hi, i tried to use the tool to backup my plex install but the metadata is not being backed up.
I will include my log,
[Backup.log](https://github.com/user-attachments/files/16154789/Backup.log)

The script also were not restarting plex but i added /machine:amd64 and it worked, but metadata folder still not on backup. Also the registry key is not present in backup.

**Comment by alekdavis** [comment]
@agent47git: You get the following error during backup of the Metadata folder:

_Exceção ao definir "LastWriteTime": "O DateTimeOffset especificado não pode ser convertido em um carimbo de data/hora de arquivo Zip. Nome do parâmetro: value" O DateTimeOffset especificado não pode ser convertido em um carimbo de data/hora de arquivo Zip. Nome do parâmetro: value_

I do not know what the error messages say, but it is caused by the default compression. I would recommend installing 7-zip and using 7-zip compression instead. I suspect it should address your issue.

**Comment by agent47git** [comment]
The error says  that the specified datetimeoffset can't be converterd to a date/hour timestamp.

I already tried using 7zip, but it gives me this error "Cannot import run-time settings from default configuration file (if any). JSON primitivo inválido: 7zip."

I just set to 7zip this way

>     
    },
    "Type": {
        "_meta": {
            "set": ",7zip,Robocopy",
            "default": ""
        },
        "value": 7zip


I always install 7zip in default location. So i'm not sure what is the issue. Maybe is windows 11, because i already used your tool before and worked flawlessly,
